step.yml

```diff
@@ -2,8 +2,47 @@ title: Gradle Unit Test
 summary: |-
   Runs Unit Tests with `gradlew`.
 description: |-
-  Runs Unit Tests with `gradlew`.
+  This Step runs unit tests with `gradlew` and exports the results of the tests in the Bitrise test results directory (`$BITRISE_GRADLE_TEST_RESULT`)
   You can specify the test tasks to run and set task flags to run specific files.
+
+  ### Configuring the Step
+
+  To use this Step, you need at least two things:
+  * [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html).
+  * A Gradle task that is correctly configured in your Gradle project.
+
+  To configure the Step:
+  1. Add the task you want to run in the **Test task** input.
+  1. Set the `gradlew` file path in the **Path to gradlew file**. The path should be relative to the project's root. 
+  1. (Optional) You can set the file path to a `build.gradle` file for the Step in the **Optional path to the gradle build file to use input**.
+  
+  To configure which files will be cached by the Step: 
```

Comment:
This option will tell which intermediate build files (downloaded / generated during the build) will be cached IF `cache-push` step is used, otherwise this has no effect. (Not this step is doing the caching)